Makefile

```diff
@@ -0,0 +1,9 @@
+DOCKER_RUN = docker-compose run --rm php
+
+vendor:
+	${DOCKER_RUN} composer install --no-interaction
+
+.PHONY: test
+test:
```

Comment:
Another DX opinion: self autonomous command. Make test should be able to be exécutée after a gît clone, and install dependencies. 
It's much easier for any newcomer.